src/model_theory/basic.lean

```diff
@@ -0,0 +1,328 @@
+/-
+Copyright (c) 2021 Aaron Anderson, Jesse Michael Han, Floris van Doorn. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Aaron Anderson, Jesse Michael Han, Floris van Doorn
+-/
+import data.nat.basic
+
+/-!
+# Basics on First-Order Structures
+This file defines first-order languages and structures in the style of the Flypitch project.
```

Comment:
```suggestion
This file defines first-order languages and structures in the style of the
[Flypitch project](https://flypitch.github.io/).
```
And I think it'd be good to cite the Flypitch papers below and add them to the bibliography.